server/parsec/components/memory/invite.py

```diff
@@ -257,7 +258,12 @@ async def _conduit_listen(
                 invitation.conduit_state == NEXT_CONDUIT_STATE[ctx.state]
                 and curr_our_payload is None
             ):
-                return ctx.peer_payload, invitation.conduit_is_last_exchange
+                # Careful here: it's possible that peer has already changed the state
+                # of the conduit by the time we reach this point. The means it might
```

Comment:
```suggestion
                # of the conduit by the time we reach this point (typically if the peer reset the conduit to state 1, hence obviously clearing `conduit_is_last_exchange` flag to `False`). The means it might
```

Reply:
I don't think this corresponds to the issue #7212. I'd rather not add more confusion here.